tensorflow_lite_support/python/task/audio/core/tensor_audio.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TensorAudio class."""
+
+import numpy as np
+
+from tensorflow_lite_support.python.task.audio.core.pybinds import _pywrap_audio_buffer
+from tensorflow_lite_support.python.task.audio.core import audio_record
+
+_CppAudioFormat = _pywrap_audio_buffer.AudioFormat
+_LoadAudioBufferFromFile = _pywrap_audio_buffer.LoadAudioBufferFromFile
+
+
+class TensorAudio(object):
+  """A wrapper class to store the input audio."""
+
+  def __init__(self,
+               audio_format: _CppAudioFormat,
+               buffer_size: int,
```

Comment:
Let's keep the init method simple. You don't need to pass the buffer here but use the load from array function. Besides, there's no use for the is_from_file parameter so let's remove it.

Reply:
@khanhlvg I've simplified the init method. Could you check it out?